hips/hip-0000.md

```diff
@@ -0,0 +1,77 @@
+---
+hip: 9999
+title: "H4HIP: Charts v3 Enablement"
+authors: [ "Matt Farina <matt.farina@suse.com>" ]
+created: "2025-01-09"
+type: "feature"
+status: "draft"
+---
+
+## Abstract
+
+This HIP proposes the creation of charts v3, updating Helm to handle charts v2 and v3, and a
```

Comment:
The only problem, is we don't quite know what v3 would look like yet I think.

So we should aim to support both v2 and a future (or "placeholder" v3?

Reply:
This HIP proposes adding a new apiVersion for charts without removing any existing versions. The design enables multiple apiVersions of charts, with significant differences, to be handled by Helm.

We must ensure existing charts continue to work so we cannot remove the old apiVersion without breaking that. If the existing code for the existing apiVersion is massively changed it will be difficult to ensure charts don't break across apiVersions.